packages/ketcher-react/src/script/editor/operations/rxn/index.ts

```diff
@@ -1,3 +1,4 @@
+import Restruct, { ReRxnArrow } from '../../../render/restruct'
```

Comment:
Please revert i: imports should follow after 'copyright' text

Reply:
Done